requirements_base.txt

```diff
@@ -73,7 +73,7 @@ parameterized==0.7.4      # via -r requirements.in
 pbr==5.4.5                # via jira
 pdfminer.six==20201018
 # pip-tools==5.1.2          # via -r requirements.in
-protobuf==3.15.0          # via -r requirements.in, google-api-core, google-cloud-bigquery, googleapis-common-protos
+protobuf==3.20.1          # via -r requirements.in, google-api-core, google-cloud-bigquery, googleapis-common-protos
```

Comment:
What triggered this requirements update?

Reply:
If you jump to the github home page for the RDR project, it is displaying a library version vulnerability alert. https://github.com/all-of-us/raw-data-repository.
This takes care of that alert.